client/src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,21 @@
+import Button from '@material-ui/core/Button';
+import useStyles from './useStyles';
+import { AppBar, Typography } from '@material-ui/core';
+import logo from '../../Images/logo.png';
+
+const NavBar = (): JSX.Element => {
```

Comment:
Also, please check with Ibsa as he has been assigned the task for creating the 'Dashboard Skeleton aka Navbar'. Just to make sure we are not duplicating components, thanks!

Reply:
sure Rajiv , let me confirm with him ...